.travis.yml

```diff
@@ -32,15 +36,16 @@ matrix:
         - php: 7.0
           env: SYMFONY_VERSION=2.8.* COVERAGE=true SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.0
-          env: SYMFONY_VERSION=3.0.*
+          env: SYMFONY_VERSION=3.2.*
 
 before_script:
-    - if [ "$COVERAGE" != "true" ] && [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi
+    - if [ "$COVERAGE" != "true" ]; then phpenv config-rm xdebug.ini; fi
     - composer self-update
     - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi
-    - composer update $COMPOSER_FLAGS --prefer-source
+    - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    - composer update $COMPOSER_FLAGS --prefer-dist --no-interaction
 
-script: if [ "$COVERAGE" == "true" ]; then ./phpunit --coverage-clover=clover; else ./phpunit; fi
+script: if [ "$COVERAGE" == "true" ]; then vendor/bin/phpunit --process-isolation --coverage-clover=clover; else vendor/bin/phpunit --process-isolation; fi
```

Comment:
Can't we do this in the `phpunit.dist.xml` file so that it also works locally ?

Reply:
I remember wondering this then postponing it, let's see.

Reply:
Even better, I think it could be done on tests that really need it with some annotation

Reply:
Pushed a new version with that